lib/sqlalchemy/engine/default.py

```diff
@@ -1181,7 +1183,9 @@ class DefaultExecutionContext(ExecutionContext):
     result_column_struct: Optional[
         Tuple[List[ResultColumnsEntry], bool, bool, bool, bool]
     ] = None
-    returned_default_rows: Optional[Sequence[Row[Any]]] = None
+    returned_default_rows: Optional[
+        Sequence[Row[Unpack[Tuple[Any, ...]]]]
```

Comment:
**Federico Caselli** (CaselIT) wrote:

maybe `util.typing` could export `UnpackAny = Unpack[Tuple[Any, ...]]` since it seems the used everywhere the generic is not needed

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
mypy doesn't allow such things. I have added `TupleAny` type.

Reply:
I kinda had this suspicion. That's still better than before, thanks!